src/modules/dashboard/components/SuggestionCreate/SuggestionCreate.tsx

```diff
@@ -82,16 +79,18 @@ const SuggestionCreate = ({ colonyAddress, domainId, openDialog }: Props) => {
       onSubmit={handleSubmit}
       validationSchema={validationSchema}
     >
-      <Input
-        appearance={{ theme: 'fat' }}
-        label={MSG.inputLabelTitle}
-        name="title"
-        innerRef={inputRef}
-      />
+      <div onFocus={handleUnclaimedProfile}>
+        <Input
```

Comment:
This is another UI change suggestion.

![Screenshot from 2020-01-22 15-19-05](https://user-images.githubusercontent.com/1193222/72898183-44c77880-3d2c-11ea-9f74-bf40facdad1e.png)

I suggest we show either a tooltip saying something like: "Press return to create a new suggestion"

Or just add a button at the end of the input, like we have on domains:

![Screenshot from 2020-01-22 15-32-42](https://user-images.githubusercontent.com/1193222/72898292-75a7ad80-3d2c-11ea-8ee3-5b1288394807.png)

Either way, I think that for "some" users the current suggestion creation flow will be confusing.

Thoughts ?

And again, maybe @JoinColony/product can chime in here...

Reply:
You're absolutely right - I think the user would benefit by having some sort of instructive text or symbol.

Another solution would be to use the `InputStatus` (just below the input) to display some text (when it's valid, of course). It's already there, so I think it'd be the easiest & fastest solution to implement.

<img width="1474" alt="Screen Shot 2020-01-22 at 8 04 37 AM" src="https://user-images.githubusercontent.com/3052635/72900673-f5625780-3ced-11ea-89b5-e0b53c054ddd.png">

Reply:
That said, I think @JoinColony/product should make this call 🙂

Reply:
I wouldn't use the status field for help text. For that we have the `help` property. Let's keep it consistent!

Reply:
We decided on the button 🙂

Reply:
Ah, even better!